src/app/saga/manuscript.saga.ts

```diff
@@ -15,6 +15,7 @@ export function* loadManuscriptSaga(action: Action<string>) {
     const manuscript = yield call(getManuscriptContent, id);
     yield put(manuscriptActions.loadManuscriptAction.success(manuscript));
   } catch (e) {
+    console.log(e);
```

Comment:
Needed?

Reply:
Oops. Sorry. I think I'm gonna add something like a debug output that prints those things  in console only when in dev mode.